CODEOWNERS

```diff
@@ -2,6 +2,10 @@
 
 /packages/dd-trace/src/appsec/ @DataDog/asm-js
 /packages/dd-trace/test/appsec/ @DataDog/asm-js
+
+/packages/dd-trace/src/debugger/ @DataDog/debugger
+/integration-tests/debugger/ @DataDog/debugger
```

Comment:
if i were you i would do this instead, otherwise you're never gonna merge anything haha
```suggestion
/packages/dd-trace/src/debugger/ @DataDog/debugger @Datadog/dd-trace-js
/integration-tests/debugger/ @DataDog/debugger @Datadog/dd-trace-js
```

Reply:
I thought the first line in the file automatically added our team to all files by using `*`?

Reply:
I don't think so no, you can double check, but I remember being in this situation and blocking myself from merging anything, but i could be wrong

Reply:
I've just verified, and it works like I expected 😀

Take a look at this line in `CODEOWNERS`:
https://github.com/DataDog/dd-trace-js/blob/99497961d3dc70b5b55eaef3d52248f93936f0ab/CODEOWNERS#L13

Here's [a 3 week old PR](https://github.com/DataDog/dd-trace-js/pull/4609) that changes a file under that directory. If you see what happened once the PR was marked as "ready for review", both `@DataDog/ci-app-libraries` and `@DataDog/dd-trace-js` was requested for a code owners review:

<img width="798" alt="image" src="https://github.com/user-attachments/assets/ab707ba7-b6ca-4d00-bb43-23f4d0c22818">

Reply:
Then what is the point of this line for example then: https://github.com/DataDog/dd-trace-js/blob/99497961d3dc70b5b55eaef3d52248f93936f0ab/CODEOWNERS#L8

Reply:
No need to have the `@Datadog/dd-trace-js` team on that line

Reply:
Nope, you're actually 100% correct. I just discovered a flaw in my logic above. The PR I linked to happened to also touch 3 files that was directly owned by `@Datadog/dd-trace-js`, hence why the PR requested a code owners review from both teams 🤦. My bad. I'll update the entry

Reply:
Also, I managed to finally track down the docs for how the `*` entry on the first line of the file should be interpreted:

```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2
```

_([source](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file))_

Reply:
Feel free to close that thread when it's fixed :+1: